src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -16,11 +18,34 @@ import { TwoColumnLayout } from "../../Components/TwoColumnLayout"
 
 export interface ReviewProps {
   order: Review_order
+  router: Router
 }
 
-export class ReviewRoute extends Component<ReviewProps> {
+interface ReviewState {
+  termsCheckboxSelected?: Boolean
+}
+
+export class ReviewRoute extends Component<ReviewProps, ReviewState> {
+  constructor(props) {
+    super(props)
+
+    this.state = { termsCheckboxSelected: false }
+  }
+
+  updateTermsCheckbox() {
+    const { termsCheckboxSelected } = this.state
+    this.setState({
+      termsCheckboxSelected: !termsCheckboxSelected,
+    })
+  }
+
+  onOrderSubmitted() {
+    this.props.router.push(`/order2/${this.props.order.id}/submission`)
```

Comment:
cc @ashfurrow I think you can add something similar to this in your mutation handler to advance to the next step.